lib/routes/content.js

```diff
@@ -437,3 +484,4 @@ const routes = application => {
 };
 
 module.exports = routes;
+module.exports.check = check;
```

Comment:
If you find yourself needing to export an additional function from these route files, it likely means it doesn't belong in this file and should be moved to a separate, related file